joomla/joomla-cms
# User Registration Captcha

Pull Request for Issue #21921 .

### Summary of Changes
Moves the captcha to the end of the form. This had already been done for contacts and content


### Testing Instructions
1. Enable recaptcha
2. set the captcha in global config
3. enable user registration
4. create custom user fields for registration
5. check the registration form on the front end - captcha displayed
6. disable captcha in global config
7. check the registration form on the front end - no captcha displayed
8. enable captcha in the users component
9.  check the registration form on the front end - captcha displayed

### Actual result BEFORE applying this Pull Request
the captcha is rendered after the default fields but before any custom fields


### Expected result AFTER applying this Pull Request
the captcha is rendered at the end of the form
